test/specs/core.interaction.tests.js

```diff
@@ -192,6 +192,47 @@ describe('Core.Interaction', function() {
 		});
 	});
 
+	describe('x-axis mode', function() {
+		var data = {
+			datasets: [{
+				label: 'Dataset 1',
+				data: [10, 20, 30],
+				pointHoverBorderColor: 'rgb(255, 0, 0)',
+				pointHoverBackgroundColor: 'rgb(0, 255, 0)'
+			}, {
+				label: 'Dataset 2',
+				data: [40, 40, 40],
+				pointHoverBorderColor: 'rgb(0, 0, 255)',
+				pointHoverBackgroundColor: 'rgb(0, 255, 255)'
+			}],
+			labels: ['Point 1', 'Point 2', 'Point 3']
+		};
+
+		beforeEach(function() {
```

Comment:
I think `data` and `acquireChart` can go directly in the `it(` until we need to refactor for another test?

Reply:
sure, will change